crates/moot/tests/moot_lmoo.rs

```diff
@@ -84,7 +84,7 @@ impl TelnetMootRunner {
     fn resolve_response(&mut self, response: String) -> Result<String, std::io::Error> {
         // Resolve the response; for example, the test assertion may be `$object`; resolve it to the object's specific number.
         self.client(WIZARD).command(format!(
-            "; {response}; \"TelnetMootRunner::resolve_response\";"
```

Comment:
Yeah this was completely busted, without the `return` we just... accepted everything :D